src/Uno.UI/UI/Xaml/DependencyPropertyDetails.cs

```diff
@@ -130,8 +130,26 @@ private void SetValueFull(object? value, DependencyPropertyValuePrecedences prec
 				return;
 			}
 
-			// If we were unsetting the current highest precedence value, we need to find the next highest
-			if (valueIsUnsetValue && precedence == _highestPrecedence)
+			// Clear the animated value, when we are setting a local value to a property
+			// with an animated value from the filling part of an HoldEnd animation.
+			// note: There is no equivalent block in SetValueFast, as its condition would never be satisfied:
+			// _stack would've been materialized if the property had been animated.
+			bool isAnimationOverriddenByLocal = false;
+			if (!valueIsUnsetValue &&
```

Comment:
So if we clear the local value, it won't erase the "animation filling value" but setting any other value will ?

Also instead of adding an external flag and having some "animation logic" here, I've the feeling that we could have added a new precedence for this value. Then the value would be set at the end of the animation and read / updated / cleared by the `Timeline` itself when needed.

Reply:
\> So if we clear the local value, it won't erase the "animation filling value" but setting any other value will ?
I'll have to validate against windows, and see what they do in that case: 
- set local, but keep animation; vs
- discard animation and find next high precedence

\> Also instead of adding an external flag and having some "animation logic" here, I've the feeling that we could have added a new precedence for this value.
I had thought about this. What I had feared was the rippling effect, so I took a shortcut. I will experiment with this approach.

Reply:
On Windows, explicitly calling SetValue(dp, DP.UnsetValue) or ClearValue(dp) doesnt clear the animation value.
Given dp default value is 0, local set to 450, animated and held at 900,
calling either of above two methods will result in:
- GetValue: 900 -> 900
- ReadLocalValue: 450 -> DP::UnsetValue
- GetAnimationBaseValue: 450 -> 0

From this we can infer that:
- animation was and still is the highest precedence
- local value was cleared, or in other words, set to UnsetValue
- GetAnimationBaseValue went from reading the local (450) to a tier below (0) which is likely dp metadata's default value, as local value is cleared